src/Components/WebView/WebView/test/Microsoft.AspNetCore.Components.WebView.Test.csproj

```diff
@@ -1,12 +1,27 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk.Razor">
 
   <PropertyGroup>
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+    <OutputType>Exe</OutputType>
     <RootNamespace>Microsoft.AspNetCore.Components.WebView</RootNamespace>
+    <!-- Suppress CS8002 because Photino assemblies are not signed -->
+    <NoWarn>@(NoWarn);CS8002</NoWarn>
+    <!-- See code comment in Program.cs/Main in this project regarding this -->
+    <StartupObject>Microsoft.AspNetCore.Components.WebView.Program</StartupObject>
   </PropertyGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\..\Samples\PhotinoPlatform\src\Microsoft.AspNetCore.Components.WebView.Photino.csproj" />
```

Comment:
It would follow the repo conventions a bit more if, instead of referencing Photino (which we don't support) from M.A.C.WebView.Test (when we do support WebView in general), there was a separate E2E test project for WebView which references Photino.

Conceptually, these new tests really are E2E, so a project structure something like this would be a perfect fit into the repo, and mirrors what we have for `Components` generally:

![image](https://github.com/dotnet/aspnetcore/assets/1101362/06c92f54-9c28-43f7-af75-532719ef526c)

I'm not meaning to just create extra work here, and am fine if you don't want to refactor Photino platform to support having test projects that are separate from Photino platform itself. But it would be preferable not to reference Photino platform from the non-E2E WebView test project, since we don't want to have possible cases where WebView unit tests start relying on things from Photino.

Reply:
Alright the new structure is much more like this now, and I agree it's cleaner.